app/controllers/system/admin/get_help_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+class System::Admin::GetHelpController < System::Admin::Controller
+  def index
+    start_date = params[:start_date].presence
+    end_date = params[:end_date].presence
+    load_users(start_date, end_date)
+    @get_help_data = fetch_all_recent_live_feedbacks(start_date, end_date)
+    load_assessment_question_hash
+    load_course_user_hash
+  end
+
+  private
+
+  def load_users(start_date = nil, end_date = nil)
+    @users = if start_date.present? || end_date.present?
+               User.human_users.includes(:emails)
+             else
+               User.human_users.includes(:emails).active_in_past_7_days
+             end
+    @user_hash = @users.index_by(&:id)
+  end
+
+  def load_course_user_hash
+    # Get all unique course IDs from the assessment data
+    course_ids = @assessment_question_hash.values.map { |v| v[:course_id] }.uniq
+
+    @course_user_hash = {}
+    course_ids.each do |course_id|
+      course_users = CourseUser.where(course_id: course_id, user_id: @user_hash.keys)
+      @course_user_hash[course_id] = course_users.index_by(&:user_id)
+    end
```

Comment:
N+1 query. Should be able to get all the `CourseUser.where(course_id: course_ids, ...)` in a single DB query.